### PR TITLE
style: update link styles to adhere to design specification

### PR DESCRIPTION
**Pull request by kyubisation**
BREAKING CHANGE: This removes the margin around the public variant of sbbLink.
If you want to retain the margin, it is `margin: 1em 0 2em;`.

### CODE REVIEW

File: src/angular-business/links/link.component.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  text-decoration: none;
+  word-break: break-word;
+
+  &:hover,
+  &:focus {
+    color: $sbbColorCallToActionHover;
+  }
+}
+
+.sbb-icon.sbb-link-business-icon {
```
Comment (kyubisation) [inline src/angular-business/links/link.component.scss:17]:
```suggestion
.sbb-link-business-icon.sbb-icon {
```

File: src/angular-core/styles/typography/_link.scss [hunk 2]
```diff
@@ -3,22 +3,10 @@
...
 @mixin standardLink {
   color: $sbbColorBlack;
-  line-height: pxToEm(23px);
+  line-height: pxToRem(23px);
   text-decoration: underline;
   text-decoration-skip-ink: auto; // -> read more about here https://medium.com/@iamhiwelo/improving-text-readability-for-dyslexic-users-with-skip-ink-underlines-bf52a2f3426b
 
-  @include publicOnly() {
-    @include mq($from: desktop4k) {
-      font-size: pxToRem($sizeFontDefault4k);
```
Comment (jeripeierSBB) [inline src/angular-core/styles/typography/_link.scss:12]: I think, we still need the font-size part here? As it is now, there is no 4k and 5k font-size

File: src/angular-public/links/link.component.scss [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    margin-left: pxToRem(8 * $scalingFactor5k);
+  }
+
+  .sbb-link:hover &,
+  .sbb-link-form & {
+    color: inherit;
+  }
+}
+
+.sbb-link-icon-left.sbb-icon {
```
[the remark this answers is not in the document]
Comment (kyubisation) [reply src/angular-public/links/link.component.scss:85]: Looking back my change to inline the variants for sbb-link might have been a mistake.
I really want as little nesting as possible.

[the remark this answers is not in the document]
Comment (jeripeierSBB) [reply src/angular-public/links/link.component.scss:85]: we need to challenge that :P ;-)

**Pull request merged**